variables.tf

```diff
@@ -0,0 +1,66 @@
+variable "env_name" {
+  default = "uapipeline-dev"
+}
+
+variable "openstack_cloud" {
+  default = ""
+}
+
+variable "pubkey" {
+  default = "~/.ssh/id_rsa.pub"
+}
+
+variable "privkey" {
+  default = "~/.ssh/id_rsa"
+}
+
+variable "postgresql_flavor" {
+  default = "medium1"
+}
+
+variable "postgresql_image" {
+  default = "ubuntu-16.04.raw"
```

Comment:
if possible I'd recommend going with 18.04 that will give you support till 2023, same for all of the images below.

Reply:
Righto. Was going to ask you about that. Thanks.

Reply:
Addressed with: https://github.com/julianpistorius/terraref-terraform/commit/d63658dbb571bd66c2564ce8d9f7f91eae286dff